org.gnome.Ptyxis.json

```diff
@@ -0,0 +1,96 @@
+{
+    "app-id" : "org.gnome.Ptyxis",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46beta",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "ptyxis",
+    "finish-args" : [
+        "--allow=devel",
+        "--device=dri",
+        "--filesystem=host",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
Please ask for an exception for this https://docs.flathub.org/docs/for-app-authors/linter#exceptions https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
we need to sort out the id first since they are tied.

Reply:
https://github.com/flathub-infra/flatpak-builder-lint/pull/339 in the mean time

Reply:
This is also resolved now that the ID issue is dealt with.